config/locales/devise.views.es.yml

```diff
@@ -120,7 +120,7 @@ es:
         sign_in_with_provider: Iniciar sesión con %{provider}
         sign_up: Registrarse
       minimum_password_length:
-        one: "(%{count} caractere como mínimo)"
+        one: "(%{count} caracter como mínimo)"
```

Comment:
Good catch! 
This is actually taken from the [devise locale](https://github.com/tigrish/devise-i18n/blob/master/rails/locales/es.yml). Maybe you can propose that change for the gem?

Reply:
That's bad Spanish though, it doesn't matter if they wrote it 😉

Reply:
Yeah I agree! 
I meant to also suggest the change to the [gem](https://app.localeapp.com/projects/377/). I've done the suggestion already 😀